WebSecurityScanner/metadata/V1Beta/Finding.php

```diff
@@ -16,9 +16,32 @@ public static function initOnce() {
         }
         \GPBMetadata\Google\Api\Resource::initOnce();
         \GPBMetadata\Google\Cloud\Websecurityscanner\V1Beta\FindingAddon::initOnce();
-        $pool->internalAddGeneratedFile(hex2bin(
-            "0ae5090a34676f6f676c652f636c6f75642f77656273656375726974797363616e6e65722f7631626574612f66696e64696e672e70726f746f1226676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e7631626574611a3a676f6f676c652f636c6f75642f77656273656375726974797363616e6e65722f7631626574612f66696e64696e675f6164646f6e2e70726f746f22a9060a0746696e64696e67120c0a046e616d6518012001280912140a0c66696e64696e675f7479706518022001280912130a0b687474705f6d6574686f6418032001280912120a0a66757a7a65645f75726c180420012809120c0a04626f647918052001280912130a0b6465736372697074696f6e18062001280912180a10726570726f64756374696f6e5f75726c18072001280912110a096672616d655f75726c18082001280912110a0966696e616c5f75726c18092001280912130a0b747261636b696e675f6964180a20012809123a0a04666f726d18102001280b322c2e676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e7631626574612e466f726d12510a106f757464617465645f6c696272617279180b2001280b32372e676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e7631626574612e4f757464617465644c69627261727912550a1276696f6c6174696e675f7265736f75726365180c2001280b32392e676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e7631626574612e56696f6c6174696e675265736f7572636512550a1276756c6e657261626c655f68656164657273180f2001280b32392e676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e7631626574612e56756c6e657261626c6548656164657273125b0a1576756c6e657261626c655f706172616d6574657273180d2001280b323c2e676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e7631626574612e56756c6e657261626c65506172616d657465727312380a03787373180e2001280b322b2e676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e7631626574612e5873733a8401ea4180010a2977656273656375726974797363616e6e65722e676f6f676c65617069732e636f6d2f46696e64696e67125370726f6a656374732f7b70726f6a6563747d2f7363616e436f6e666967732f7b7363616e5f636f6e6669677d2f7363616e52756e732f7b7363616e5f72756e7d2f66696e64696e67732f7b66696e64696e677d4294020a2a636f6d2e676f6f676c652e636c6f75642e77656273656375726974797363616e6e65722e763162657461420c46696e64696e6750726f746f50015a58676f6f676c652e676f6c616e672e6f72672f67656e70726f746f2f676f6f676c65617069732f636c6f75642f77656273656375726974797363616e6e65722f7631626574613b77656273656375726974797363616e6e6572aa0226476f6f676c652e436c6f75642e57656253656375726974795363616e6e65722e563142657461ca0226476f6f676c655c436c6f75645c57656253656375726974795363616e6e65725c563162657461ea0229476f6f676c653a3a436c6f75643a3a57656253656375726974795363616e6e65723a3a563162657461620670726f746f33"
-        ), true);
+        $pool->internalAddGeneratedFile(
```

Comment:
this looks.... wrong.

@dwsupplee @bshaffer

Reply:
It functions correctly, and certainly does look odd. It is related to the following: https://github.com/protocolbuffers/protobuf/pull/8006

I'm not a huge fan of the approach, but I think we're okay to move forward with it right now.